build-tools/scripts/RunTests.targets

```diff
@@ -190,6 +201,7 @@
          Projects="$(MSBuildThisFileDirectory)RunTests.targets"
          RunEachTargetSeparately="True"
          Targets="@(_RunTestTarget)"
+         Properties="%(Properties)"
```

Comment:
It will be batched AFAIK.

I wrote a small test to show it.

```
<Project DefaultTargets="MyBuild" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <ItemGroup>
    <TestTarget Include="CallMe1">
      <Property>MyProperty=prop1</Property>
    </TestTarget>
    <TestTarget Include="CallMe2">
      <Property>MyProperty=prop2</Property>
    </TestTarget>
  </ItemGroup>

  <Target Name="CallMe1">
    <Message Text="CallMe1 $(MyProperty)"/>
  </Target>

  <Target Name="CallMe2">
    <Message Text="CallMe2 $(MyProperty)"/>
  </Target>

  <Target Name="MyBuild">
    <MSBuild
        Projects="test1.csproj"
	RunEachTargetSeparately="True"
	Targets="@(TestTarget)"
	Properties="%(Property)" />
  </Target>
</Project>
```
Result
```
msbuild test1.csproj
Microsoft (R) Build Engine version 16.0.42-preview+g804bde742b for Mono
Copyright (C) Microsoft Corporation. All rights reserved.

Build started 4/17/2019 5:00:44 PM.
Project "/Users/rodo/git/xa-master/test1.csproj" on node 1 (default targets).
Project "/Users/rodo/git/xa-master/test1.csproj" (1) is building "/Users/rodo/git/xa-master/test1.csproj" (1:2) on node 1 (CallMe1 target(s)).
CallMe1:
  CallMe1 prop1
Done Building Project "/Users/rodo/git/xa-master/test1.csproj" (CallMe1 target(s)).
Project "/Users/rodo/git/xa-master/test1.csproj" (1) is building "/Users/rodo/git/xa-master/test1.csproj" (1:3) on node 1 (CallMe2 target(s)).
CallMe2:
  CallMe2 prop2
Done Building Project "/Users/rodo/git/xa-master/test1.csproj" (CallMe2 target(s)).
Done Building Project "/Users/rodo/git/xa-master/test1.csproj" (default targets).

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:00.37
```

Reply:
If you use `%(Foo)` in the same task as a `@(Bar)`, it will assume you are using the `Foo` item metadata from `Bar`.

However, I do think it is clearer if you put `%(Bar.Foo)`, @radekdoulik maybe that would make things more clear?

Reply:
Yup, in this case it might help. Note that in other cases where there are at least 2 item groups it would change the process as it would batch only the explicitly named item group.